sample/src/screens/EndToEndTestsScreen.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect} from 'react';
+import {ScrollView, Text, Platform} from 'react-native';
+
+import * as Sentry from '@sentry/react-native';
+
+import {getTestProps} from '../../utils/getTestProps';
+import {SENTRY_INTERNAL_DSN} from '../dsn';
+
+const EndToEndTestsScreen = () => {
+  const [eventId, setEventId] = React.useState(null);
+
+  // !!! WARNING: Do not put Sentry.init inside React.useEffect like we do here. This is only for testing purposes.
```

Comment:
I know it's a sample, but if it's not supposed to do the init here, why do we do it? maybe an explanation/comment.

Reply:
We use it just to use `setEventId` inside `beforeSend` to get the event id. Let me add a comment.